### DISCUSSION TITLE
Using a newer version of Strapi

### DISCUSSION THREAD

**Discussion by jvzaniolo** [SGFGOV/medusa-strapi-repo#79] [Q&A]
What would be the easiest option to use a newer version of Strapi with the Medusa plugin?
- Create a new Strapi project, add the `strapi-plugin-medusajs` and create the Collections by hand
- Or upgrade the medusa-strapi project dependencies

**Comment by SGFGOV** [comment]
Hi We don't support strapi v5

Otherwise..you could just fork the repo and you are good to go

Best regards
Govind

On Thu, 18 Apr, 2024, 7:09 pm João Vitor, ***@***.***> wrote:

> What would be the easiest option to use a newer version of Strapi with the
> Medusa plugin?
>
>    - Create a new Strapi project, add the strapi-plugin-medusajs and
>    create the Collections by hand
>    - Or upgrade the medusa-strapi project dependencies
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SGFGOV/medusa-strapi-repo/discussions/79>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHDZUO4PGFKKC5JB6IDY57EINAVCNFSM6AAAAABGNJV46GVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGUZDIMRRGU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.
